src/jaguar.toit

```diff
@@ -201,28 +225,49 @@ uninstall_image name/string -> none:
     else:
       logger.error "container '$name' not found"
 
-run_code image_size/int reader/reader.Reader defines/Map -> none:
-  timeout/Duration? := null
+compute_timeout defines/Map disabled/bool -> Duration?:
```

Comment:
Make the boolean a named argument.

Alternatively: you already get the `defines`.
So you can also look into it to see if it has `JAG_DISABLED`. If yes, then it needs a default timeout.

Reply:
Done.